source/content/environment-specific-config.md

```diff
@@ -36,22 +35,25 @@ if (!defined('WP_DEBUG') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
 }
 ```
 
-<Alert title="Warning" type="danger">
-
-PHP constants like `WP_DEBUG` can only be defined once. When implementing this code snippet, remove or comment out the [existing code block](https://github.com/pantheon-systems/WordPress/blob/master/wp-config.php#L147) defining it.
+PHP constants like `WP_DEBUG` can only be defined once. When implementing this code snippet, change the [existing code block](https://github.com/pantheon-systems/WordPress/blob/default/wp-config.php#L72-L74) defining it:
 
-</Alert>
+```php:title=wp-config.php
+if ( ! defined( 'WP_DEBUG' ) ) {
+    define('WP_DEBUG', (wp_get_environment_type() == "development"));
+}
+```
```

Comment:
So, I didn't notice before, but an equivalent example based on `$_ENV['PANTHEON_ENVIRONMENT']` exists just above here, lines 29-35 (collapsed by default). Maybe we should remove the existing example, and only show the one based on wp_get_environment_type().

Reply:
I think the latest commit resolves this cc @carolyn-shannon for copy

Reply:
Sorry to catch this only after PR merged, but `wp_get_environment_type()` cannot be called in wp-config.php, as the rest of WordPress has not yet loaded. Opened #5922.